src/calendar/export/CalendarParser.ts

```diff
@@ -266,72 +267,59 @@ function parseAlarm(alarmObject: ICalObject, event: CalendarEvent): AlarmInfo |
 	}
 	const triggerValue = triggerProp.value
 	if (typeof triggerValue !== "string") throw new ParserError("expected TRIGGER property to be a string: " + JSON.stringify(triggerProp))
-	let trigger: AlarmInterval
+	let alarmInterval: AlarmInterval
 
 	// Absolute time
 	if (triggerValue.endsWith("Z")) {
+		// For absolute time we just convert the trigger to minutes. There might be a bigger unit that can express it but we don't have to take care about time
```

Comment:
can we normalize zero-length intervals to always use the same unit?

Reply:
I guess we could. Is that something we want to do?

Reply:
It's probably not important since we don't deduplicate them anymore anyway